.changeset/eight-hairs-run.md

```diff
@@ -0,0 +1,6 @@
+---
+'lit': minor
+'lit-html': minor
+---
+
+Add `isServer` environment checker
```

Comment:
Maybe a little more detail so that it reads nicely in the CHANGELOG and on Twitter? Something like:

Adds an `isServer` variable export to `lit` and `lit-html/is-server.js` which will be `true` on Node and `false` in the browser. This can be used when authoring components to change behavior based on whether or not the component is executing in an SSR context.

Reply:
Great suggestion. Done!